rules/sinks/third_parties/sdk/amazon/javascript.yaml

```diff
@@ -193,3 +193,859 @@ sinks:
     tags:
+
+  - id: ThirdParties.SDK.Amazon.Aws.S3
+    name: Amazon Aws S3
+    domains:
+      - "aws.amazon.com/s3"
+    patterns:
+      - "@aws-sdk\\/client-s3"
```

Comment:
I see some rules already present can we delete duplicate rules.

Reply:
removed